pages/ref/index.mdx

```diff
@@ -0,0 +1,59 @@
+# Reference overview
+
+import { Cards, Steps } from 'nextra/components'
+
+Welcome to the **Language** section of Tact documentation — a place for becoming more familiar with the language in much more depth compared to the [Book](/book).
+
+Here are its main contents:
+
+<Steps>
+
+### API Reference
```

Comment:
I don't really understand why this is called "API Reference". Is it the Standard library reference? To me this sounds like the reference of the TypeScript API the Tact compiler provides

Reply:
Fixed by renaming it to "Core library".

I think analogy of it being a one big chunk of code divided into categories (or sub-pages) works with the fact that it's auto-included and doesn't require an explicit `import` statement like "Standard libraries"